### DISCUSSION TITLE
how to use build_production?

### DISCUSSION THREAD

**Discussion by anlexN** [maizzle/framework#622] [Q&A]
- Maizzle Version: 4.0.0-alpha.1
- Node.js Version: v17.1.0
![image](https://user-images.githubusercontent.com/8686739/153329235-3781afc7-0bc6-489b-bcca-980cf51ddb9a.png)

**Comment by cossssmin** [comment]
You use the output HTML in your email service provider, like MailChimp for example. Most ESPs provide a way for you to define a template by pasting in HTML code.

**Comment by anlexN** [comment]
so how to use images folder?

**Comment by cossssmin** [comment]
You upload them to your email service provider or to a CDN and update the paths in the HTML.

Moving this to Discussions since it's not an issue with the framework :)
